### PR TITLE
Fix/5707  correct author when creating file from model

### PR DESCRIPTION
**Pull request by inkhey**
related to #5707 

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified
- [x] Original authors of the features included in a multi-feature branch (maintenance fixes -> develop, security fixes -> develop, …) should be part of the reviewers, especially if you encountered merge conflicts.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

### CODE REVIEW

File: backend/tracim_backend/lib/core/content.py [hunk 1]
```diff
@@ -452,43 +460,28 @@ def create(
         content.workspace = workspace
         content.type = content_type.slug
         content.is_temporary = is_temporary
-        content.revision_type = ActionDescription.CREATION
-        content.content_namespace = content_namespace
-
         if template_id:
```
Comment (grignards) [inline backend/tracim_backend/lib/core/content.py:463]: Couldn't those lines be part of `copy_as_template`?

File: backend/tracim_backend/lib/core/content.py [hunk 2]
```diff
@@ -452,43 +460,28 @@ def create(
...
-                tags_values = tag_lib.get_all(content_id=template_id)
+        return content
 
-                for tag in tags_values:
-                    tag_lib.add_tag_to_content(
-                        user=self._user, content=content, tag_name=tag.tag_name
-                    )
+    def add_tags_from_template(
+        self, content: Content, template_id: int,
+    ):
```
Comment (grignards) [inline backend/tracim_backend/lib/core/content.py:478]:
```suggestion
    def copy_tags(
        self, destination: Content, source_content_id: int,
    ) -> None:
```

File: backend/tracim_backend/lib/core/content.py (hunk 1 above)
Comment (inkhey) [reply backend/tracim_backend/lib/core/content.py:463]: probably. done.

File: backend/tracim_backend/lib/core/content.py (hunk 2 above)
Comment (inkhey) [reply backend/tracim_backend/lib/core/content.py:478]: done.

**Comment by gbonaspetti** [comment]
Tested on branch

**Pull request merged**